charts/hono/templates/artemis/artemis-deployment.yaml

```diff
@@ -30,6 +30,7 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      priorityClassName: "{{ $args.componentConfig.pod.priorityClassName }}"
```

Comment:
I think it would be better to put this in a
`{{- if $args.componentConfig.pod.priorityClassName }}`
block.

And, the quoting is usually done via the `quote` template function in the Hono chart:
`priorityClassName: {{ $args.componentConfig.pod.priorityClassName | quote }}`

Reply:
As this is repeated often, a named template in `_helpers.tpl` could be added.